packages/js/src/Modules/Verto/webrtc/BaseCall.ts

```diff
@@ -752,6 +756,26 @@ export default abstract class BaseCall implements IWebRTCCall {
     this.setBandwidthEncodingsMaxBps(max, 'video');
   }
 
+  /**
+   * Registers callback for stats.
+   * 
+   * @param callback 
+   * @param constraints 
+   * @returns 
+   */
+  getStats(callback: Function, constraints: any) {
+    if (!callback) {
+      return;
+    }
+    const binding: IStatsBinding = { callback: callback, constraints: constraints };
+    this._statsBindings.push(binding);
+  
+    if (!this._statsIntervalId) {
+      const STATS_INTERVAL = 5000;
```

Comment:
What do you think about the interval being able to be configurable also @piotrgregor?

Reply:
Absolutely, yes. I'll add this to interface IStatsBinding packages/js/src/Modules/Verto/webrtc/interfaces.ts as optional param @carloscarvallo (if no interval is configured, default will be used, but I'll change default to 2 seconds)